pmd-compat6/src/it/pmd-for-java/pom.xml

```diff
@@ -21,6 +21,18 @@
                 <artifactId>maven-pmd-plugin</artifactId>
                 <version>@maven-pmd-plugin.version.for.integrationtest@</version>
                 <executions>
+                    <execution>
+                        <id>java-processing-error-pmd</id>
+                        <goals>
+                            <goal>pmd</goal>
+                        </goals>
+                        <phase>verify</phase>
+                        <configuration>
```

Comment:
This configuration section is not applied due to https://issues.apache.org/jira/projects/MPMD/issues/MPMD-236 : the check goal actually calls the pmd goal, but it doesn't apply the configuration for the specific execution.

It's easier, if we keep only one execution here and add the rulesets to the main configuration (lines 31/43 below):

```xml

                    <rulesets>
                        <ruleset>/rulesets/java/maven-pmd-plugin-default.xml</ruleset>
                        <ruleset>${project.basedir}/exception_ruleset.xml</ruleset>
                    </rulesets>
```

Note: it explicitly uses the default maven-pmd-plugin ruleset plus the additional one to provoke a processing error.